doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,6 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For a sqlite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Set \"serverVersion\" to your server version to avoid edge-case exceptions and extra database calls",
-        "DATABASE_URL": "mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
+        "DATABASE_URL": "mysql://app:app@127.0.0.1:3306/app?charset=utf8mb4&serverVersion=5.7"
```

Comment:
A real-world example:

`pgsql://%env(DB_USER)%:%env(DB_PASSWORD)%@%env(DB_HOST)%:%env(DB_PORT)%/%env(DB_DBNAME)%?charset=UTF8&serverVersion=9.6`

While I appreciate that using a URL / DSN provides the greatest flexibility, it's actually not useful in most cases because as the comment at the top of `etc/container.yaml` rightly points out:

```yaml
# Put parameters here that don't need to change on each machine where the app is deployed
# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```

A typical user wouldn't use e.g. a different type and version of DBMS in different deployment environments.

(For testing, this is already covered by setting `APP_ENV=test` and overriding in `etc/packages/test/doctrine.yaml`.)